frontend/src/components/Navigator/index.js

```diff
@@ -703,6 +712,8 @@ Navigator.propTypes = {
       PropTypes.string,
     ]),
   }),
+  socketMessageStore: PropTypes.shape({
+  }),
```

Comment:
could we update this to represent an object of shape `user: { name: string }`, unless there are additional properties I missed from `SocketAlert`?